hbp_knowledge/pp2a/seshacharyulu2013.bel

```diff
@@ -203,80 +242,118 @@ a(CHEBI:cantharidin) -| act(p(FPLX:PPP2))
 a(PUBCHEM:"4162") -| act(p(FPLX:PPP2))
 a(PUBCHEM:"6913994") -| act(p(FPLX:PPP2))
 a(CHEBI:Tautomycin) -| act(p(FPLX:PPP2))
+UNSET Confidence
 
 SET Evidence = "In addition to microbial toxin, viral protein SV40 (potent
 oncogene) also inactivates PP2A action by binding to the AC dimer and displacing the PR56
 (PP2A-B’ γ ) subunit [35]."
+SET Confidence = "High"
 a(PUBCHEM:"71361092") -> path(MESH:Neoplasms)
 a(PUBCHEM:"71361092") -| act(p(FPLX:PPP2))
+UNSET Confidence
 
 SET Evidence = "Endogenous CIP2A (cancerous inhibitor of PP2A) inhibits PP2Ac
 activity via interacting with c-Myc (Ser62) and stabilizes it from proteolytic degradation
 [49]. "
+SET Confidence = "Medium"
+p(HGNC:CIP2A) -> composite(p(HGNC:CIP2A),p(HGNC:MYC))
+composite(p(HGNC:CIP2A),p(HGNC:MYC)) -| deg(p(HGNC:MYC))
+p(HGNC:MYC) -| act(p(FPLX:PPP2C))
 p(HGNC:CIP2A) -| act(p(FPLX:PPP2C))
+UNSET Confidence
 
 SET Evidence = "Recently, a major indirect activation of PP2A by inhibiting CIP2A at both the
 transcriptional and translational levels through the drug bortezomib was shown in triple
 negative breast cancer cells [50]."
+SET Confidence = "Medium"
+SET MeSHDisease = "Breast Neoplasms"
 a(CHEBI:bortezomib) -| p(HGNC:CIP2A)
+a(CHEBI:bortezomib) -| r(HGNC:CIP2A)
 a(CHEBI:bortezomib) -> act(p(FPLX:PPP2C))
+UNSET MeSHDisease
+UNSET Confidence
 
 SET Evidence = "Methylation [52] and phosphorylation [53] are two major
 modifications that have been shown to modulate PP2A catalytic efficiency."
+SET Confidence = "High"
 p(FPLX:PPP2, pmod(M)) -- act(p(FPLX:PPP2))
 p(FPLX:PPP2, pmod(Ph)) -- act(p(FPLX:PPP2))
+UNSET Confidence
 
+#can be annotated with respect to shi2009 subunit classification
 SET Evidence = "Phosphorylation of Y307 by
 receptor associated tyrosine kinases effectively decreases the PP2A activity by inhibiting the
 interaction of PP2Ac with the PP2A-PR55/PR61 subunit [54]."
+SET Confidence = "Low"
 p(FPLX:PPP2, pmod(Ph,Tyr,307)) -| act(p(FPLX:PPP2))
+#p(FPLX:PPP2, pmod(Ph,Tyr,307)) -| complex(p(HGNC:PPP2R2A),p(HGNC:PPP2R5B))
+UNSET Confidence
 
+#annotation of the regulatory and core dimers
 SET Evidence = " The addition of a methyl group by LCMT1 at L309 enhances the binding affinity
 of the core dimer (A&C subunit) toward distinct regulatory subunits and provides specific
 activity to the holoenzyme [56]."
+SET Confidence = "Low"
```

Comment:
i think it's not really possible to code such a complex process, also because the distinct regulatory subunits are not mentioned with names. 
That is why I just coded the final result, the activation.
If you want, you could go back to the original paper to maybe learn more about the process.. then an additional statement could be added to show the different steps of the process